### DISCUSSION TITLE
Edit the author

### DISCUSSION THREAD

**Discussion by xcy7e** [PaulWoitaschek/Voice#1161] [Ideas]
Would be awesome to have the possibility to edit the author of an audiobook.
Currently I need to change the author in the audiofile itself before moving it to the phone's storage for Voice to read it in.

If there is a wrong author, changing the Information in the audiofile afterwards does not affect the voice storage, for good reasons I think. Anyway, not bring able to change author makes it a bit more difficult to manage the audiobook collections.

**Comment by ahstro** [comment]
Technically, #1407 is kind of a duplicate of this, buut it has more upvotes, is slightly more comprehensive, and has maintainer response, so maybe close this? 🤷
